grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -676,7 +676,7 @@ class OrderService {
                     if (quantity == "") {
                         throw new IllegalArgumentException("Missing quantity.")
                     }
-                    Integer parsedQty = Integer.valueOf(quantity)
+                    Integer parsedQty = java.text.NumberFormat.instance.parse(quantity)
```

Comment:
you can add an import instead of using package here, also you can use the getNumberInstance() instead of instance, like it's done in all other places